src/hooks/useLoadGlobalLateData.js

```diff
@@ -27,7 +31,12 @@ export default function useLoadGlobalLateData() {
     const promises = [];
     const p1 = loadWalletBalanceNamesToCache();
     const p2 = dispatch(nonceManagerLoadState());
-    promises.push(p1, p2);
+    const p3 = ensIntroMarqueeNames.map(async name => {
+      prefetchENSResolveName(name);
```

Comment:
prefetching is happening here (on app init)

Reply:
i think we removed this before? https://github.com/rainbow-me/rainbow/pull/3116#discussion_r872838992

Reply:
what about adding it here? https://github.com/rainbow-me/rainbow/blob/develop/src/screens/WalletScreen.js#L167 only if `walletIsReady`

Reply:
I thought this was more global data, and not account data (the you linked to was account related).

Reply:
actually, you're right, probs better to put in WalletScreen